hack/docker/voyager/templates/http-frontend.cfg

```diff
@@ -12,6 +12,11 @@ frontend {{ .FrontendName }}
 	option httplog
 	option forwardfor
 
+    {{- range $rule := .FrontendRules }}
+    {{ $rule -}}
+    {{ end }}
+
```

Comment:
`{{range pipeline}} T1 {{else}} T0 {{end}}`

Should we not inject any custom rule when Frontend Rules are used. Then user has full control.
```
    {{- range $rule := .FrontendRules }}
    {{ $rule }}
    {{ else }}
 	option httplog
  	option forwardfor
    {{ end }}
```